griptape/drivers/prompt/cohere_prompt_driver.py

```diff
@@ -52,12 +52,18 @@ def _prompt_stack_input_to_message(self, prompt_input: PromptStack.Input) -> dic
     def _base_params(self, prompt_stack: PromptStack) -> dict:
         user_message = prompt_stack.inputs[-1].content
 
-        history_messages = [self._prompt_stack_input_to_message(input) for input in prompt_stack.inputs[:-1]]
+        history_messages = [
+            self._prompt_stack_input_to_message(input) for input in prompt_stack.inputs[:-1] if input.content
```

Comment:
Can you compare to `None` explicitly with `input.content is not None`?

Reply:
Sadly, no. it's a problem with it being an empty string. So I can check for None or an empty string, but that's what the boolean check is doing.

```
'chat_history': [{'role': 'SYSTEM', 'text': ''}]}
```

Reply:
Ah, gotcha, nevermind then.